kpatch-build/kpatch-build

```diff
@@ -831,6 +834,23 @@ fi
 
 [[ -n "$OOT_MODULE" ]] || grep -q vmlinux "$SRCDIR/Module.symvers" || die "truncated $SRCDIR/Module.symvers file"
 
+while read -ra sym_line; do
+    if [[ ${#sym_line[@]} != 4 ]]; then
+        die "Malformed ${TEMPDIR}/Module.symvers file"
+    fi
+
```

Comment:
Checkout the symvers_read() function in lookup.c and it documents some of the various Module.symvers formats we've seen over the years.  I think you'll need to be more flexible than checking for exactly four columns.